cmd/snapd-env-generator/main.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include<stdlib.h>
```

Comment:
Please run `make fmt` here. The indenting is rather unexpected.

Reply:
Sure, done. Note that only a single line changed:
```
-        const char *snap_bin_dir = SNAP_MOUNT_DIR"/bin";
+       const char *snap_bin_dir = SNAP_MOUNT_DIR "/bin";
```

Reply:
What? did indent now decide to `#include<foo>`? Oh the horror :)